### PR TITLE
1 ➡️ 2

### PR DESCRIPTION
**Pull request by mjcarroll**
# ➡️ Forward port

Port `ign-utils1` to `main`

Branch comparison: https://github.com/ignitionrobotics/ign-utils/compare/main...ign-utils1

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

### CODE REVIEW

File: BUILD.bazel [hunk 1]
```diff
@@ -18,9 +18,9 @@ exports_files(["LICENSE"])
 
 PROJECT_NAME = "ignition-utils"
 
-PROJECT_MAJOR = 2
+PROJECT_MAJOR = 1
```
Comment (azeey) [inline BUILD.bazel:21]: Do we want this change?

File: CMakeLists.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
 #============================================================================
 # Initialize the project
 #============================================================================
-project(ignition-utils2 VERSION 2.0.0)
+project(ignition-utils1 VERSION 1.2.0)
```
Comment (azeey) [inline CMakeLists.txt:6]: I don't think we want this change

File: CMakeLists.txt [hunk 3]
```diff
@@ -15,7 +15,7 @@ find_package(ignition-cmake2 2.0.0 REQUIRED)
 # Configure the project
 #============================================================================
 set(c++standard 17)
-ign_configure_project(VERSION_SUFFIX pre1)
+ign_configure_project(VERSION_SUFFIX)
```
Comment (azeey) [inline CMakeLists.txt:18]: I don't think we want this change

File: Changelog.md [hunk 4]
```diff
@@ -1,8 +1,18 @@
-## Ignition Utils 2.x
+## Ignition Utils 1.x
```
Comment (azeey) [inline Changelog.md:1]: Can you keep the `2.x` section?

**Review by azeey — CHANGES_REQUESTED** [review]

**Comment by mjcarroll** [comment]
Did this one too quickly, also waiting on #37

**Review by azeey — APPROVED** [review]

**Review by chapulina — APPROVED** [review]

**Pull request merged**